$NetBSD: patch-utilbin_Makefile,v 1.1 2011/02/17 08:55:11 markd Exp $

replace $(COMMLISTSLIB) with -lcommlists so that pkgsrc wrapper
doesnt reorder.

--- source/utilbin/Makefile.orig	2011-02-10 02:13:18.354047083 +0000
+++ source/utilbin/Makefile
@@ -282,7 +282,7 @@ sge_share_mon.o: $(UTILBIN_DIR)/sge_shar
 		$(CC) $(CFLAGS) -c $(UTILBIN_DIR)/sge_share_mon.c
 
 #---------------------------------------------------------------------
-AUTHUSER_ADD_LIBS = -luti -llck -lrmon $(COMMLISTSLIB) $(DLLIB)
+AUTHUSER_ADD_LIBS = -luti -llck -lrmon -lcommlists $(DLLIB)
 
 authuser: authuser.o juti.o $(UTILIB) $(LCKLIB) $(RMONLIB) $(COMMLISTSLIB)
 	$(CC) -o authuser $(LFLAGS) authuser.o juti.o $(AUTHLIBS) $(AUTHUSER_ADD_LIBS) $(LIBS)
